packages/mobile/src/components/drawer/NativeDrawer.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react'
+
+import { SetOptional } from 'type-fest'
+
+import { useNativeDrawer } from 'app/hooks/useDrawer'
+import { Drawer as DrawerName } from 'app/store/drawers/slice'
+
+import Drawer, { DrawerProps } from './Drawer'
+
+type NativeDrawerProps = SetOptional<DrawerProps, 'isOpen' | 'onClose'> & {
+  drawerName: DrawerName
+}
+
+export const NativeDrawer = (props: NativeDrawerProps) => {
```

Comment:
Would be helpful to have a component comment  that explains NativeDrawer vs. Drawer vs. AppDrawer
/**
 *
 */